packages/packages/api/src/medical/models/patient.ts

```diff
@@ -52,6 +12,7 @@ export type PatientUpdate = z.infer<typeof patientUpdateSchema>;
 
 export const patientSchema = patientUpdateSchema.extend({
   facilityIds: z.array(z.string()),
+  links: linkStatusAcrossHIEsSchema,
```

Comment:
Why do we need `links` in the patient update? Updating a patient's ID/demo is currently separate from any notion of links